builder/utils/deployer.py

```diff
@@ -114,7 +114,7 @@ def _save_task_info_json(self, task_code):
 
     def setup_sagemaker_env(self):
         env = {}
-        region = self.model.task.aws_region
+        region = self.config.get("sagemaker_region") or self.model.task.aws_region
```

Comment:
I think the easier fix for this issue of not being able to specify region is just adding a UI field for the admin page for a task to specify the aws_region for a task (it might be confusing to have one field in the build_config and one field determined by the task aws_region).

There is already a field in the DB, so its just a matter of adding it to https://github.com/facebookresearch/dynabench/blob/main/frontends/web/src/containers/TaskOwnerPage.js
and
https://github.com/facebookresearch/dynabench/blob/main/frontends/web/src/components/TaskOwnerPageComponents/Settings.js

(you can find the relevant places to add to the UI by searching everywhere one of the other fields is used e.g. `build_sqs_queue`: https://github.com/facebookresearch/dynabench/search?q=build_sqs_queue). If this is too much, no worries we can create an issue for it and I can try and take it on later

The reason we didn't do this before was because we have just had everything in us-west-1 so there's been no reason to change the aws_region.

Reply:
I think it's better to ignore task.aws_region. The evaluation server should be the one deciding where/how to run the evaluation. Currently only Flores tasks where using this column, because of the proto-decentralized implementation I made.